layouts/event/single.html

```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
+            <div><strong>{{ T "license" }} </strong> <a
+                    href="{{ T (printf "lic-url-%s" ($.Scratch.Get "license")) }}">{{ T (printf "lic-%s" ($.Scratch.Get "license")) }}</a>
+                <span class="hidden" xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Dataset"
+                    property="dct:title" rel="dct:type">{{ .Title }}</span>
+                     </div>{{ end }}
```

Comment:
What is this `{{ end }}` tag closing? This should lead to a build error.

```suggestion
                     </div>
```